package.json

```diff
@@ -96,7 +96,7 @@
     "supertest": "^4.0.2"
   },
   "mocha": {
-    "timeout": 5000,
+    "timeout": 10000,
```

Comment:
Tests error pretty often in CI because of timeouts. Haven't had it happen locally, so I blame slow CI servers :smile: